src/coordinator/users.spec.js

```diff
@@ -850,4 +850,135 @@ describe('Coordinator/Users', function() {
       });
     });
   });
+
+  describe('subscribeEvent', function() {
+    context('when all the required parameters are provided', function() {
+      let expectedUserEvent;
```

Comment:
Maybe make this `expectedSubscription`? `expectedUserEventSubscription` is getting pretty long

Reply:
👍